.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 build
 coverage.xml
+.idea
```

Comment:
This shouldn't be in here. Put your IDEs meta folder in your global gitignore, please.

Reply:
What's the issue with it being here, @mfn?

Reply:
It doesn't make sense to have an ever expanding list of all the editors/IDEs in each code repository people work on. Next one wil add `.vscode`, then `.sublime` or `.eclipse` etc.

These should be in each developers global ignore.

Reply:
What's wrong with that? There aren't really that many IDEs, I think it's fairly standard to have that in .gitignore.